css/style.css

```diff
@@ -454,6 +454,113 @@ textarea {
   margin: 8px auto;
 }
 
+.modal {
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  z-index: 99999;
+  background-color: hsl(216, 14%, 79%, 0.8);
+  backdrop-filter: blur(4px);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  overflow-y: scroll;
+}
```

Comment:
- [ ] The modal works as expected, but the mobile view looks broken. Please make sure to render the mobile view of the modal as provided in Figma: 

Your design  | Figma design
------------- | -------------
<img width="325" alt="Screenshot 2022-03-09 at 18 46 48" src="https://user-images.githubusercontent.com/77038610/157465150-b6234bf5-d067-45ed-8c7f-21adc066d6dc.png"> | <img width="285" alt="Screenshot 2022-03-09 at 18 46 08" src="https://user-images.githubusercontent.com/77038610/157465021-ac0d4476-d078-4737-b4c3-eed118096da6.png">

More specifically: 

- [ ] Please make sure that the content fits the modal, currently, buttons and text are going out of modal white background (if the mobile screen size is 320px). 
- [ ] Please render only 3 technologies `HTML, CSS, javascript` in mobile view.